test/lib/Analysis/TestAllocation.cpp

```diff
@@ -37,23 +37,22 @@ struct TestAllocationPass
   }
 
   void runOnOperation() override {
-    auto &os = llvm::errs();
     ModuleOp moduleOp = getOperation();
     // Convert to std::string can remove quotes from opName
     ModuleAllocation moduleAllocation = getModuleAllocation();
     moduleOp.walk([&](triton::FuncOp funcOp) {
       auto opName = SymbolTable::getSymbolName(funcOp).getValue().str();
-      os << opName << "\n";
+      mlir::emitRemark(funcOp.getLoc(), opName);
```

Comment:
Just curious, is preferred to using remark in mlir tests? If so, should we change other places with remark as well?

Reply:
It makes the test much more robust because the remarks are tied to the op location.

Reply:
I don't know what all the print-based tests are, but I can do a quick survey, and yeah many print tests could use MLIR diagnostics to be more robust.

Reply:
I see TestAlias and TestAxisInfo. Let me send a separate PR for updating those.